project/build.scala

```diff
@@ -306,7 +306,7 @@ object Dependency {
     // runtime deps versions
     val Chaos = "0.8.7"
     val Guava = "19.0"
-    val MesosUtils = "1.0.0-rc1"
+    val MesosUtils = "1.0.0-rc2"
```

Comment:
Please add a comment here (regarding deprecation) as well, so we get reminded if we change the version here.